### ISSUE TITLE
cant import tags from Tasks.org app 

### ISSUE THREAD

**Issue by Pleasantless** [etesync/etesync-web#187]
any tags added using the Android app [Tasks.org](https://play.google.com/store/apps/details?id=org.tasks&hl=en&gl=US) are not imported by EteSync. it isnt a sync issue because the following works well:
_ tags/tasks created on web to app
_ tasks edited on web to app
_ tasks edited on app imported to web

**Comment by tasn** [comment]
What you are describing about empty tags sounds a bit like #179.
Tasks created on Tasks.org: can you please give an example raw task here? Open the web app -> collections -> the tasks collection -> click on the task -> change history -> raw content and copy it here (feel free to censor the summary line). I wanna see how the tags are synced for you.

I'm not quite sure I understand what doesn't work. So the only thing that doesn't work is that tags created with Tasks.org are not show on EteSync Web for you? Though tags are synced the other way?

**Comment by Pleasantless** [comment]
yes, #179 is something im facing when i create a tag using the web (though not always).
but the 2nd issue is that tags created using the 3rd party Android app (Tasks.org) do not even sync in the first place (though tasks do sync)

**Comment by tasn** [comment]
Fixed #179. I wonder if your other issue is related to what we just fixed with Android version 2.2.0.

**Comment by Pleasantless** [comment]
still not fixed for me.

**Comment by tasn** [comment]
OK, but at least #179 is fixed?

**Comment by Pleasantless** [comment]
well, fixed as in no new tags can be added.

**Comment by tasn** [comment]
I just tried it and it works well. I think maybe the task needs to become a bubble after typing it (so maybe you need to click space, or return or something)?

**Comment by Pleasantless** [comment]
oh, you were correct. but no tasks sync from the app to web still :(
